namesilo.php

```diff
@@ -2305,28 +2305,25 @@ private function manageDnsRecords(
 
         if (!empty($post)) {
             if (isset($post['action'])) {
+                $dns_fields = [
```

Comment:
This causes an error when deleting a record since fields like 'record_type' are not defined

Reply:
Updated to only get those fields on add or update